app/src/main/java/ar/com/wolox/android/example/utils/Extras.kt

```diff
@@ -14,4 +15,15 @@ object Extras {
         const val USERNAME = "username"
         const val PASSWORD = "password"
     }
+
+    object MockTesting {
+        const val INVALID_EMAIL = "camilo.ortiz"
+        const val FAIL_EMAIL = "camilo.ortiz@wolox.co"
+        const val FAIL_PASSWORD = "123"
+        val FAIL_USERDATA = LoginData(FAIL_EMAIL, FAIL_PASSWORD)
+
+        const val EMAIL = "clinton.harris59@example.com"
+        const val PASSWORD = "123456"
+        val USERDATA = LoginData(EMAIL, PASSWORD)
+    }
```

Comment:
This is okay for now, but in real projects it's better to create a specific file for each tests' data.